website/docs/r/securitycenter_subscription_pricing.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_securitycenter_subscription_pricing"
+sidebar_current: "docs-azurerm-securitycenter-subscription-pricing"
+description: |-
+    Manages the subscription's Security Center pricing tier.
```

Comment:
I think this'd read clearer as `Manages the Pricing Tier for Azure Security Center in the current subscription` - what do you think (and below too)